### ISSUE TITLE
Cannot access logName when implementing FileLoggerOptions.FormatLogEntry

### ISSUE THREAD

**Issue by PaulBol** [nreco/logging#15]
Great project!

I only noticed one minor issue when trying to add some additional information to the log output like the thread ID.

We can set `FileLoggerOptions.FormatLogEntry` to customize the text that is written to the log file.

The original code in `Log<TState>` includes `logBuilder.Append(logName)`. When using `FormatLogEntry` it doesn't seem possible to read `logName` in order to write it to the text file. Or am I missing anything?

**Comment by VitaliyMF** [comment]
Yes, you're right `FormatLogEntry` cannot get `logName` in the current implementation (in FileLogger.cs):
```
			if (LoggerPrv.FormatLogEntry!=null) {
				LoggerPrv.WriteEntry(LoggerPrv.FormatLogEntry(
					new LogMessage(logLevel, eventId, message, exception)));
			}
```
However it is very easy to fix that, it is enough to add missed properties to `LogMessage` class and populate them in constructor  -- this will be fully backward-compatible change. 

Feel free to submit PR for that if you want to get this change ASAP :)

**Comment by VitaliyMF** [comment]
Shipped in 1.1.1.
Thanks for the PR!
